lib/boring_generators/cli.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+require "rails/generators"
+require_relative "../boring_generators"
+
+module BoringGenerators
```

Comment:
If possible, can you check if we can add specs for this? I am okay to ship without specs as long as this is working. We can always come back and increase coverage. 

We can add spec in the existing generator spec file as well.

Reply:
@abhaynikam Of course, I'll add unit test for cli.